### PR TITLE
Upgrade to rails 6

### PR DESCRIPTION
**Pull request by kkoehn**
Upgrade Rails
Have not seen any breaking changes

requires 
ruby_version     = ">= 2.5.0"
rubygems_version = ">= 1.8.11"

### CODE REVIEW

**Comment by MrSerth** [comment]
I checked the requirements on our server, we fulfill these ;).
So there should be nothing to do besides triggering a rollout once we merged.

File: config/initializers/new_framework_defaults_6_0.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 6.0 upgrade.
```
Comment (MrSerth) [inline config/initializers/new_framework_defaults_6_0.rb]: Do we need to continue working on some of these default settings (for future Rails upgrades) or is everything done?

File: db/schema.rb [hunk 2]
```diff
@@ -173,6 +173,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "file_extension"
+    t.string "editor_mode"
```
Comment (MrSerth) [inline db/schema.rb]: We might merge #238 first

File: config/initializers/new_framework_defaults_6_0.rb (hunk 1 above)
Comment (kkoehn) [reply config/initializers/new_framework_defaults_6_0.rb]: You are right, we shouldn't need this file anymore.

Comment (MrSerth) [reply config/initializers/new_framework_defaults_6_0.rb]: 👍

**Pull request merged**